src/config/SpreedlyApiConfigt.ts

```diff
@@ -0,0 +1,27 @@
+import { AxiosInstance } from 'axios';
+import { ISpreedlyConfiguration } from '../interface';
+import { createAxiosInstance } from '../utils';
+
+export class SpreedlyApiConfig {
```

Comment:
I think that `Spreedly*` prefix is redundant, as we already are in the Spreedly library